sites/example-project/src/pages/api/status/[...route]/extractQueries.server.js

```diff
@@ -0,0 +1,124 @@
+import unified  from "unified";
+import remarkParse from "remark-parse";
+import { visit } from "unist-util-visit";
+import md5 from "blueimp-md5";
+import fs from "fs-extra";
+
+const strictBuild = (process.env.VITE_BUILD_STRICT === 'true')
+const circularRefErrorMsg = 'Compiler error: circular reference'
+
+// Unified parser step to ignore indented code blocks. 
+// Adapted from the mdsvex source, here: https://github.com/pngwn/MDsveX/blob/master/packages/mdsvex/src/parsers/index.ts
+// Discussion & background here:  https://github.com/evidence-dev/evidence/issues/286
+const ignoreIndentedCode = function() {
+	const Parser = this.Parser;
+	const block_tokenizers = Parser.prototype.blockTokenizers;
+	block_tokenizers.indentedCode = () => true;
+}
+
+const updateDirectoriesandStatus = function (queries, routeHash) {
+  let queryDir = `./.evidence-queries/extracted/${routeHash}`;
+
+  fs.ensureDirSync(queryDir);
+
+  const existingQueries = fs.readJSONSync(`${queryDir}/queries.json`, {
+    throws: false,
+  });
+
+  queries.forEach((query) => {
+    query.queryStringMD5 = md5(query.compiledQueryString);
+    if(existingQueries){
+        let existingQuery = existingQueries.find(
+            (existing) =>
+              existing.id === query.id &&
+              existing.queryStringMD5 === query.queryStringMD5
+          );
+          if (existingQuery) {
+            query.status = existingQuery.status;
+          } else {
+            query.status = "not run";
+          }
+    }else{
+        query.status = "not run";
+    }   
+  });
+
+  if (queries.length === 0) {
+    fs.emptyDirSync(queryDir);
+  } else {
+    fs.writeJSONSync(`${queryDir}/queries.json`, queries);
+  }
+
+  let status = queries.map((query) => {
+    return { id: query.id, status: query.status };
+  });
+
+  return status;
+};
+
+export const getStatusAndExtractQueries = function (route) {
+  let routeHash = md5(route);
+  let content = fs.readFileSync(`./src/pages/${route}/+page.md`);
```

Comment:
From what I can see it's failing when it's looking on path
```
  path: './src/pages/settings/+page.md
```
or 
```
  path: './src/pages/status/+page.md'
```

Those folders contain `.svelte` instead of `.md`

(I am not super familiar with this script) Maybe this script should check for `.md` or `.svelte` ? 🤔

Reply:
Good shout, this should actually not be called at all on those settings routes.